front/app/components/Unauthorized/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+// hooks
+import useAuthUser from 'hooks/useAuthUser';
+import Button from 'components/UI/Button';
+
+// events
+import { openSignUpInModal } from 'events/openSignUpInModal';
+
+// styling
+import { colors, fontSizes } from 'utils/styleUtils';
+import { useTheme } from 'styled-components';
+
+// components
+import { Box, Title, Text } from '@citizenlab/cl2-component-library';
+
+// i18n
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+import pageNotFoundMessages from '../PageNotFound/messages';
+
+const Unauthorized = () => {
+  const theme = useTheme();
+  const { formatMessage } = useIntl();
+  const authUser = useAuthUser();
+  const authUserPending = authUser === undefined;
+
+  if (authUserPending) return null;
```

Comment:
Perhaps a Spinner here could be better than null but I'n not sure if there isn't a spinner on this screen already coming from somewhere else.